tests/acl/test_acl.py

```diff
@@ -104,25 +106,55 @@ def setup(duthosts, rand_one_dut_hostname, tbinfo, ptfadapter):
         A Dictionary with required test information.
 
     """
-    duthost = duthosts[rand_one_dut_hostname]
 
-    mg_facts = duthost.get_extended_minigraph_facts(tbinfo)
+    mg_facts = rand_selected_dut.get_extended_minigraph_facts(tbinfo)
+    topo = tbinfo["topo"]["type"]
+
+    vlan_ports = []
+    vlan_mac = ""
+
+    if topo == "t0":
+        vlan_ports = [mg_facts["minigraph_ptf_indices"][ifname]
+                      for ifname in mg_facts["minigraph_vlans"].values()[0]["members"]]
+
+        config_facts = rand_selected_dut.get_running_config_facts()
+        vlan_table = config_facts["VLAN"]
+        vlan_name = list(vlan_table.keys())[0]
```

Comment:
In the case when there are multiple vlans, this would always get only one first vlan from the list. We have started adapting testcases to cover multiple vlan scenario, like here - https://github.com/Azure/sonic-mgmt/issues/3625

Should test_acl be adapted accordingly?

Reply:
Supporting multiple VLANs would require significant code changes I think. We can address this in a later PR, but for now I'd prefer to get this change merged so we can improve the test results.